Formula/llvm.rb

```diff
@@ -130,6 +130,7 @@ class Llvm < Formula
   option "with-python", "Build bindings against custom Python"
   option "with-shared-libs", "Build shared instead of static libraries"
   option "without-libffi", "Do not use libffi to call external functions"
+  option "without-experimental", "Do not build libc++experimental library"
```

Comment:
Given we don't actually use this libc++ ourselves at all I'm tempted to say this should be enabled with `with-libcxx` (so by default). Thoughts?